deploy/k8s/charts/trustification/templates/services/report/060-Deployment.yaml

```diff
@@ -60,10 +60,10 @@ spec:
         - name: config-volume
           configMap:
             name: {{ include "trustification.common.name" $mod }}-nginx-config
-        {{- if $mod.module.enabled_auth}}
-        - name: htpasswd
-          configMap:
-            name: {{ include "trustification.common.name" $mod }}-report-htpasswd
+        {{- if $mod.module.enabledAuth}}
+        - name: auth-secret-volume
+          secret:
+            secretName: report-auth-secret
```

Comment:
Why can't we add the prefix, like with the config map?

Reply:
Document container‘ port is 8080, if this one is 8080, we can not access the service.
secret-report-auth?

Reply:
but those run on different pods. so there should be no conflict.

Reply:
I test it, if it's 8080, I can not access report server.

Reply:
I am not sure what's wrong then. You'd need to investigate. We already have this pattern and it works:

https://github.com/trustification/trustification/blob/bdf9fca358a71b7e22be155c19b7f9955c311d07/deploy/k8s/chart/templates/docs/030-Deployment.yaml#L36-L38

https://github.com/trustification/trustification/blob/bdf9fca358a71b7e22be155c19b7f9955c311d07/deploy/k8s/chart/templates/spog/ui/030-Deployment.yaml#L66-L68

So I believe it should work.

Reply:
I made some changes with the latest commit. So 8080 should work. Additionally, I tried to simplify nginx config and leave only necessary config. And moved the image config to the values file.

Reply:
I'm wondering further if we could use "trust" image to run this as it should have nginx in it? Or is it a good idea to do that anyways?

Reply:
Might make sense, yes.